api/handlers.go

```diff
@@ -0,0 +1,57 @@
+package api
+
+import (
+	"net/http"
+
+	"github.com/SkynetLabs/malware-scanner/database"
+	"github.com/julienschmidt/httprouter"
+	"gitlab.com/NebulousLabs/errors"
+	skyapi "gitlab.com/SkynetLabs/skyd/node/api"
+)
+
+// healthGET returns the status of the service
+func (api *API) healthGET(w http.ResponseWriter, r *http.Request, _ httprouter.Params) {
+	status := struct {
+		DBAlive     bool `json:"dbAlive"`
+		ClamAVAlive bool `json:"clam_av_alive"`
```

Comment:
nit: why the deviation from camelCase?

Reply:
Ha! Some auto-complete stuff slipped through. Thanks.